aten/src/ATen/cudnn/Descriptors.cpp

```diff
@@ -126,7 +126,7 @@ void FilterDescriptor::set(const at::Tensor &t, int64_t pad) {
   }
   dim = std::max(dim, pad);
   cudnnTensorFormat_t filter_format = CUDNN_TENSOR_NCHW;
-  if (t.suggest_memory_format() == at::MemoryFormat::ChannelsLast) {
+  if (t.suggest_memory_format(/*channels_last_strides_exact_match=*/true) == at::MemoryFormat::ChannelsLast) {
```

Comment:
We can't set mark `channels_last_strides_exact_match` as constant `true`.
This means we won't be able to distinguish the ambiguous case for N111 weight.